sql/init.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE `rabbit_messages` (
```

Comment:
Also an update is that we don't need `exchangeName` key, `channelId` and `deliveryTag` can be a multi column key and we would need an index for both of the new columns.